src/rf/apps/workers/image_validator.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.conf import settings
+
+import os
+import uuid
+from boto.s3.key import Key
+from boto.s3.connection import S3Connection
+from osgeo import gdal
```

Comment:
Consider ordering and grouping imports according to https://www.python.org/dev/peps/pep-0008/#imports